packages/plugins/rubygem-fog-proxmox/rubygem-fog-proxmox.spec

```diff
@@ -1,115 +1,89 @@
-Requires: %{?scl_prefix}rubygem(fog-json) >= 1.2
-Requires: %{?scl_prefix}rubygem(fog-json) < 2
-Requires: %{?scl_prefix}rubygem(ipaddress) >= 0.8
-Requires: %{?scl_prefix}rubygem(ipaddress) < 1
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 2.5
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.5
+BuildRequires: ruby >= 2.5
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 %description
 This library can be used as a module for `fog`.
 
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+%setup -q -n  %{gem_name}-%{version}
 
 %build
 # Create the gem as gem install only works on a gem file
-%{?scl:scl enable %{scl} - << \EOF}
-gem build %{gem_name}.gemspec
-%{?scl:EOF}
+gem build ../%{gem_name}-%{version}.gemspec
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
-%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
-%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -pa .%{gem_dir}/* \
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{_bindir}
-cp -pa .%{_bindir}/* \
+cp -a .%{_bindir}/* \
         %{buildroot}%{_bindir}/
+
 find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 
 %files
 %dir %{gem_instdir}
 %exclude %{_bindir}/console
 %exclude %{_bindir}/setup
 %exclude %{gem_instdir}/.bundle
-%exclude %{gem_instdir}/.codeclimate.yml
 %exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
-%exclude %{gem_instdir}/.gitlab-ci.yml
 %exclude %{gem_instdir}/.rubocop.yml
+%exclude %{gem_instdir}/.rubocop_todo.yml
 %exclude %{gem_instdir}/.ruby-gemset
 %exclude %{gem_instdir}/.solargraph.yml
-%exclude %{gem_instdir}/.travis.yml
 %exclude %{gem_instdir}/.vscode
-%exclude %{gem_instdir}/bin/setup
-%exclude %{gem_instdir}/bin/console
 %license %{gem_instdir}/LICENSE
+%exclude %{gem_instdir}/bin
 %{gem_libdir}
+%exclude %{gem_instdir}/tasks
 %exclude %{gem_cache}
 %{gem_spec}
 
 %files doc
 %doc %{gem_docdir}
-%{gem_instdir}/Gemfile
-%exclude %{gem_instdir}/Gemfile.lock
+%doc %{gem_instdir}/CHANGELOG.md
+%exclude %{gem_instdir}/Gemfile
 %doc %{gem_instdir}/README.md
-%{gem_instdir}/Rakefile
+%exclude %{gem_instdir}/Rakefile
 %doc %{gem_instdir}/docs
-%doc %{gem_instdir}/examples
-%{gem_instdir}/fog-proxmox.gemspec
+%{gem_instdir}/examples
```

Comment:
examples was flagged as doc before, is this change intended?

Reply:
No, good catch. Updating.